view/perspective/app.js

```diff
@@ -96,10 +96,21 @@ function setupSocketIOClient(persBody) {
   const namespace = u.getNamespaceString(persBody);
 
   /*
+   * if the transprotocol is set, initialize the socketio client with
+   *  the transport protocol options. The transProtocol variable is set in 
+   *  perspective.pug
+   */
+  const options = {};
+  const clientProtocol = transProtocol; 
+  if (clientProtocol !== 'undefined') {
+    options.transports = clientProtocol.replace(/\s*,\s*/g, ',').split(',');
```

Comment:
Can you add a comment about this regex and the format the socket.io options.transports requires?